src/NewTodo.js

```diff
@@ -0,0 +1,122 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import classNames from 'classnames';
+
+class NewTodo extends React.PureComponent {
+  state = {
+    users: this.props.user,
+    userID: 0,
+    taskDescription: '',
+    taskValidation: 1,
+    userValidation: 1,
+    value: '',
+  }
+
+  selectUser = (event) => {
+    const user = this.state.users.find(el => el.name === event.target.value);
+
+    this.setState(({ userID, userValidation, value }) => ({
+      userID: user.id,
+      userValidation: 1,
+      value: user.name,
+    }));
+  }
+
+  taskTitle = (event) => {
```

Comment:
this is not a verb as well, see prev comments

Reply:
fixed